File: easy.server.demo/easy.server.tomcat.demo/src/main/java/io/github/jinlongliao/easy/server/demo/logic/annotation/MsgId.java
package io.github.jinlongliao.easy.server.demo.logic.annotation;

/**
 * 消息ID
 *
 * @author: liaojinlong
 * @date: 2023/1/31 22:17
 */
public enum MsgId {
    /**
     * "测试Groovy+缓存"
     */
    TEST0("94", "Same95 测试Groovy+缓存"),
    TEST1("95", "测试Groovy+缓存"),
    TEST2("99", "测试Groovy"),
    ;
    private final String logicId;
    private final String desc;

    MsgId(String logicId, String desc) {
        this.logicId = logicId;
        this.desc = desc;
    }

    public String getLogicId() {
        return logicId;
    }

    public String getDesc() {
        return desc;
    }

    @Override
    public String toString() {
        return "MsgId{" +
                "logicId=" + logicId +
                ", desc='" + desc + '\'' +
                '}';
    }
}

File: easy.server.demo/easy.server.kotlin.demo/src/main/kotlin/io/github/jinlongliao/easy/server/kotlin/demo/logic/annotation/MsgId.kt
package io.github.jinlongliao.easy.server.kotlin.demo.logic.annotation

/**
 * 消息ID
 *
 * @author: liaojinlong
 * @date: 2023/1/31 22:17
 */
enum class MsgId(val logicId: String, val desc: String) {
    /**
     * "测试Groovy+缓存"
     */
    TEST0("94", "Same95 测试Groovy+缓存"),
    TEST1("95", "测试Groovy+缓存"),
    TEST2("99", "测试Groovy"),
    TEST3("100", "测试Cache");

    override fun toString(): String {
        return "MsgId{" +
                "logicId=" + logicId +
                ", desc='" + desc + '\'' +
                '}'
    }
}
